source/_docs/configuration/devices.markdown

```diff
@@ -3,16 +3,16 @@ title: "Adding devices to Home Assistant"
 description: "Steps to help you get your devices in Home Assistant."
 ---
 
-Home Assistant will be able to automatically discover many devices and services available on your network if you have [the discovery component](/integrations/discovery/) enabled (the default setting).
+Home Assistant will be able to automatically discover many devices and services
+available on your network.
 
-See the [integrations overview page](/integrations/) to find installation instructions for your devices and services. If you can't find support for your favorite device or service, [consider adding support](/developers/add_new_platform/).
+See the [integrations overview page](/integrations/) to find installation
+instructions for your devices and services. Most integration can be
+fully configured via the user interface these days; however, some older or
```

Comment:
Is this true? are >50% using config flow now?

Reply:
From the actual user perspective, I consider that to be true. All common stuff uses it nowadays, which results in most installations having UI configured entries for the larger part.